deta/base.py

```diff
@@ -46,6 +50,12 @@ def append(self, value: typing.Union[dict, list, str, int, float, bool]):
 
     def prepend(self, value: typing.Union[dict, list, str, int, float, bool]):
         return self.Prepend(value)
+    
+    def measureSize(self, measurable) -> int:
```

Comment:
should be `snake_case` `measure_size`

Reply:
shouldn't be in `Util` class, it's only for base

Reply:
Gotcha, thanks!